webclient/lokalita/models.py

```diff
@@ -83,11 +83,13 @@ def check_pred_archivaci(self):
         """
         Metóda na kontrolu prerekvizit pred archivací:
 
+            kontrola jako před odesláním a navíc
+
             všechny pripojené dokumenty jsou archivované.
 
             všechny DJ mají potvrzený pian
         """
-        result = []
+        result = self.check_pred_odeslanim()
```

Comment:
@jhavrlant Tady bude třeba ještě o něco důkladnější revize, protože se mi zdá, že funkce obsahuje jen kopii z akce, což ale z velké části nedává smysl (moc nechápu, jaktože to funguje). Co je problém:
- [ ] jsou chybně nastavené required_fields (neodpovídají tomu, co je ve formuláři)
- [ ] k lokalitě vůbec nemusí být nálezová zpráva - obecně tam žádný dokument nevyžadujeme
- [ ] [zde](https://github.com/ARUP-CAS/aiscr-webamcr/blob/cd0a5596773af7c7bf5edddba3c8647f1b088f13/webclient/lokalita/models.py#L182) je myslím nesmyslně pojmenovaný logger - s PIAN nemá nic společného, testuje se, že jsou ok dokumenty (to možná bude i u akce)